pkg/unvanquished_src.dpkdir/bots/subroutine_fight_or_flight_humans.bt

```diff
@@ -3,6 +3,7 @@ selector
 	sequence
 	{
 		condition alertedToEnemy
+		condition !haveWeapon( WP_HBUILD )
```

Comment:
There is `condition haveWeapon( WP_HBUILD )` in the following block so something doesn't make sense here.

Reply:
yes